cyprecice/cyprecice.pyx

```diff
@@ -985,12 +988,11 @@ cdef class Interface:
         if len(values) == 0:
             size = 0
 
-
         cdef np.ndarray[int, ndim=1] _vertex_ids = np.ascontiguousarray(vertex_ids, dtype=np.int32)
         cdef np.ndarray[double, ndim=1] _values = np.ascontiguousarray(values, dtype=np.double)
         
-        assert(_values.size == size)
-        assert(_vertex_ids.size == size)
+        assert _values.size == size, "Scalar data is not provided for all vertices in write_block_scalar_data. Check length of input data provided"
```

Comment:
```suggestion
        assert _values.size == size, "Scalar data is not provided for all vertices in write_block_scalar_data. Check length of input data provided. Provided size: {}, Expected size: {}".format(_values.size, size)
```
I think it would be helpful to also provide the values that the assertion compares.